resources/portal/templates/en/messages/collaborator_invitation_email.mjml

```diff
@@ -9,9 +9,12 @@
   <mj-section>
     <mj-column>
       <mj-text font-weight="bold" font-size="24px" padding="20px">Join {{ .AppName }} as a collaborator</mj-text>
-      <mj-text font-size="16px" padding="20px">You have been invited to join {{ .AppName }} as a collaborator. Please click the link below to view the invitation.</mj-text>
+      <mj-text font-size="16px" padding="20px">You have been invited to join {{ .AppName }} as a collaborator to manage the [project] account in Authgear. Please click the link below to view the invitation.</mj-text>
```

Comment:
`[project]` actually means substituting the app ID here. 😂 

`{{ .AppName }}` does not appear after `join` too.

Another thing to put attention is that we want to see the app ID here. Please double check if `{{ .AppName }}` refers to the app ID or the app name. We have both. The app name is defined in the translation.json under the key `app.name`.

Reply:
Currently the render uses project id, pasted an example for reference. Also fixed `[project]` 🤦